packages/graphql-hooks/rollup.config.js

```diff
@@ -14,5 +15,12 @@ export default generateRollupConfig({
     format: 'cjs',
     indent: false
   },
-  plugins: [esbuild(), babel(), sizeSnapshot()]
+  plugins: [
+    esbuild(),
+    replace({
```

Comment:
I can't find reference to `process.env.NODE_ENV` outside of some configuration files, is this vestigial?